### PR TITLE
[Py] Modify default paths

### PR DESCRIPTION
**Pull request by tsocha**
- Update defaults in setup.py
- Update defaults in tox.ini

### CODE REVIEW

File: python/setup.py [hunk 1]
```diff
@@ -24,14 +24,15 @@
 __version__ = '0.2.0'
 
 PYNGRAPH_SOURCE_DIR = os.path.abspath(os.path.dirname(__file__))
+USER_HOME_DIR = os.environ.get('HOME')
```
Comment (postrational) [inline python/setup.py]: Instead of defining `USER_HOME_DIR` here, we should add a variable such as `DEFAULT_NGRAPH_DIST_DIR`

**Comment by DawnStone** [comment]
TESTNOW

**Comment by tsocha** [comment]
TESTNOW

**Comment by Krovatkin** [comment]
@tsocha We need @postrational to approve this PR before we can merge. I'll remind him.

**Comment by postrational** [comment]
We'll need to merge https://github.com/NervanaSystems/ngraph/pull/863 first.

**Pull request merged**